docs/index.md

```diff
@@ -2,25 +2,33 @@
   <img width="100" height="100" src="https://github.com/landing-ai/landingai-python/raw/main/assets/avi-logo.png">
 </p>
 
-# Welcome to Landing.ai SDK documentation
+# Welcome to the Landing AI Python SDK Documentation
 
-Landing.ai SDK is a set of tools to help you build computer vision applications, including
-integration with [LandingLens](https://app.landing.ai/).
+The Landing AI Python SDK is a set of tools to help you build computer vision applications, including
+integrations with [LandingLens](https://app.landing.ai/).
 
-The SDK includes helper features to acquire, process and detect things in your images and videos, with the minimum lines of code possible.
+The SDK includes features to acquire, process, and detect objects in your images and videos, with the least amount of code as possible.
 
-## Quick start
+## Quick Start
+After you have trained and deployed a computer vision model in LandingLens, you can start detecting objects in your images using the Landing AI Python SDK.
 
-
-### Instalation
-First, install the SDK:
+### Install
+First, install the Landing AI Python library:
 
 ```bash
 pip install landingai~=0.3.0
 ```
 
-### Acquiring your first images
-After installing, you can start acquiring images from one of our image sources.
+???+ note
+
+    Or to get the latest version, run:
+```bash
+pip install landingai
+```
```

Comment:
I think it's better to not add this note, as it might be confusing.

Python developers are usually aware that this `~=0.3.0` above is the version, and that omitting the version number will installs the latest one (which we don't want to encourage too much, as we might eventually need to break backwards compatibility and the current documentation is not prepared to handle versioning yet).